src/game_renderer/GameRenderer.cpp

```diff
@@ -186,9 +187,33 @@ void GameRenderer::RenderIt(GLFWwindow* window,
   game_state->SetItemsInView(
       GameRenderer::GetObjectsInView(vfplane, level->getTree()));
 
+  // large far for sexy looks
+  P->popMatrix();
+  P->pushMatrix();
+  P->perspective(45.0f, aspect, 0.01f, 1000.0f);
```

Comment:
it's not but the perspective matrix is used to determine the far plane and render the skybox and I want the far plane to closer when the view frustum culling happens so that we don't render to many things from the spatial data stucture